assert/assertions_test.go

```diff
@@ -1369,6 +1370,25 @@ func TestDirExists(t *testing.T) {
 	True(t, DirExists(mockT, "../_codegen"))
 }
 
+func TestEqualFileContent(t *testing.T) {
+	mockT := new(testing.T)
+	f, err := ioutil.TempFile("", "testify-test-equal-file-content-*")
+	if err != nil {
+		t.FailNow()
+	}
+	defer f.Close()
+
+	fc := `Foo Bar
+Example file content
+`
+
+	f.Write([]byte(fc))
+
+	f.Close()
+
+	False(t, EqualFileContent(mockT, f.Name(), fc))
```

Comment:
Shouldn't we be asserting `True()` here given that we just wrote `fc` to the file?